Makefile

```diff
@@ -23,9 +23,11 @@ deps:
 	go get -u golang.org/x/net/context
 	go get -u golang.org/x/net/context/ctxhttp
 	go get -u github.com/golang/protobuf/proto
-	go get -u github.com/golang/protobuf/protoc-gen-go
+	go get -d github.com/golang/protobuf/protoc-gen-go
 	go get -d docker.io/go-docker
 	go get -d github.com/jackspirou/syscerts
```

Comment:
Why do we need `go get` here at all? Could we use `go mod download` instead? 

If `go get` is needed I would suggest to use `-d` to get reproducible dependencies.